app/src/components/evm-debugger/LogsAndReturn.tsx

```
@@ -0,0 +1,168 @@
+      </div>
+      <div class="border-b border-gray-200 dark:border-gray-800">
+        <div class="flex">
+          <button
+            onClick={() => setActiveTab("logs")}
+            class={`px-4 py-2 text-sm font-medium transition-colors ${
+              activeTab() === "logs"
+                ? "border-b-2 border-indigo-500 text-indigo-600 dark:text-indigo-400"
+                : "text-gray-500 hover:text-gray-700 dark:text-gray-400 dark:hover:text-gray-300"
+            }`}
+            aria-label="Show logs"
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)